core/domain/user_jobs_one_off.py

```diff
@@ -626,3 +628,69 @@ def map(item):
     def reduce(key, values):
         values.sort()
         yield (key, values)
+
+
+class ProfilePictureAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that verifies various aspects of profile_picture_data_url in the
+    UserSettingsModel.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        # We can raise the number of shards for this job, since it goes only
+        # over one type of entity class.
+        super(ProfilePictureAuditOneOffJob, cls).enqueue(job_id, shard_count=32)
```

Comment:
FWIW another limitation in increasing shards is not overloading the max payload for the hash tasks post-map and pre-sort/shuffle. I find that large map outputs can overwhelm that with too many shards, but this job will almost certainly not hit those limits. (Just wanted to mentioned that there's more to consider than just the number of entities).